src/components/Dropdown/index.jsx

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { useState, useCallback } from "react";
+import * as _ from "lodash";
+
+type Props = {
+  button: string,
+  label: string,
+  items: Array<Object>
+};
+
+export default function Dropdown(props: Props) {
+  const { button, label, items } = props;
+  const [isExpanded, setIsExpanded] = useState(false);
+  const [selectedItem, setSelectedItem] = useState(button);
+  const expandMenu = useCallback(() => {
+    setIsExpanded(s => !s);
+  }, [setIsExpanded]);
+  const selectItem = useCallback(
+    item => {
+      setIsExpanded(s => !s);
+      setSelectedItem(item.label);
```

Comment:
You need to set selected VALUE, and then find matching item based on the value and render it's label.